tests/test_2022_05_17.py

```diff
@@ -1,14 +1,114 @@
+
+        update_lido_app['new_address'] = lido_tx.contract_address
+        update_lido_app['execution_layer_rewards_vault_address'] = execution_layer_rewards_vault_tx.contract_address
+        update_nos_app['new_address'] = nos_tx.contract_address
+        update_oracle_app['new_address'] = oracle_tx.contract_address
+
+        return {
+            'lido': lido_tx.contract_address,
+            'nos': nos_tx.contract_address,
+            'oracle': oracle_tx.contract_address,
+            'el_rewards_vault': execution_layer_rewards_vault_tx.contract_address
+        }
+    else:
+        return { # Hardcode contract addresses here
+            'lido': '0xb16876f11324Fbf02b9B294FBE307B3DB0C02DBB',
+            'nos': '0xbb001978bD0d5b36D95c54025ac6a5822b2b1Aec',
+            'oracle': '0x7FDef26e3bBB8206135071A52e44f8460A243De5',
+            'el_rewards_vault': '0x94750381bE1AbA0504C666ee1DB118F68f0780D4'
+        } if network_name() in ("goerli", "goerli-fork") else {
+            'lido': '',
+            'nos': '',
+            'oracle': '',
+            'el_rewards_vault': ''
+        }
+
+
+lido_app_id = '0x3ca7c3e38968823ccb4c78ea688df41356f182ae1d159e4ee608d30d68cef320'
+lido_app_version = (3, 0, 0)
+
+nos_app_id = '0x7071f283424072341f856ac9e947e7ec0eb68719f757a7e785979b6b8717579d'
+nos_app_version = (3, 0, 0)
+
+oracle_app_id = '0x8b47ba2a8454ec799cd91646e7ec47168e91fd139b23f017455f3e5898aaba93'
+oracle_app_version = (3, 0, 0)
+
+oracle_contract_version = 3
+
+permission_elrewards_vault = Permission(entity='0x2e59A20f205bB85a89C53f1936454680651E618e',  # Voting
+                                        app='0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84',  # Lido
+                                        role='0x9d68ad53a92b6f44b2e8fb18d211bf8ccb1114f6fafd56aa364515dfdf23c44f')
+permission_stake_control = Permission(entity='0x2e59A20f205bB85a89C53f1936454680651E618e',  # Voting
+                                      app='0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84',  # Lido
+                                      role='0xa42eee1333c0758ba72be38e728b6dadb32ea767de5b4ddbaea1dae85b1b051f')
+
+if network_name() in ("goerli", "goerli-fork"):
+    permission_elrewards_vault = Permission(entity='0xbc0B67b4553f4CF52a913DE9A6eD0057E2E758Db',  # Voting
+                                            app='0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F',  # Lido
+                                            role='0x9d68ad53a92b6f44b2e8fb18d211bf8ccb1114f6fafd56aa364515dfdf23c44f')
+    permission_stake_control = Permission(entity='0xbc0B67b4553f4CF52a913DE9A6eD0057E2E758Db',  # Voting
+                                          app='0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F',  # Lido
+                                          role='0xa42eee1333c0758ba72be38e728b6dadb32ea767de5b4ddbaea1dae85b1b051f')
+
+mev_limit_points = 2
+max_staking_limit = 150_000 * 10 ** 18
+staking_limit_increase = 23.4375 * 10 ** 18
```

Comment:
Maybe it's worth to add some comments here, it's not obvious where this numbers comes from.